src/views/pages/ImportKeypair/index.js

```diff
@@ -1,9 +1,90 @@
-import React from 'react';
+import React, {useState} from 'react';
+import IdentitySubNav from '../../components/IdentitySubNav';
+import PageTitle from '../../components/PageTitle';
+import Button from '../../components/Button';
+import {useNavigate} from 'react-router-dom';
 
 function ImportKeypair() {
```

Comment:
Importing should be as easy as running the same RPC call that generates the key and getting the response back.

Reply:
@Romulus10 I thought we had an option of JSON or Seed Phrase here. Should I remove those options?

Reply:
@Romulus10 Removed